server/tserver/src/main/java/org/apache/accumulo/tserver/InMemoryMap.java

```diff
@@ -202,10 +203,26 @@ public InMemoryMap(Map<String,Set<ByteSequence>> lggroups, boolean useNativeMap,
     this.memDumpDir = memDumpDir;
     this.lggroups = lggroups;
 
-    if (lggroups.size() == 0)
+    if (lggroups.size() == 0) {
       map = newMap(useNativeMap);
-    else
+      mapType = useNativeMap ? "NativeMapWrapper" : "DefaultMap";
```

Comment:
I just saw that `mapType()` was invoked in the test and assumed that there was a corresponding assertion on the value at some point. Thanks for making these constants.